CHANGELOG.md

```diff
@@ -0,0 +1,54 @@
+# Changelog
+
+## Version 2.0.0
+
+### Changes to the SAML integration
+
+#### Defaults to the SHA-256 hash signature hash
+
+This module is now using SHA-256 hashing algorithm by default. SHA-1 is no
+longer recommended for ongoing use.
+
+If you wish to keep the SHA-1 algorithm, you will need to add an override
+to the `mysite/_config/saml.yml`:
+
+```
+SAMLConfiguration:
+ Security:
+   signatureAlgorithm: "http://www.w3.org/2000/09/xmldsig#rsa-sha1"
+```
+
+Otherwise the ADFS settings for the relying party trust needs to be changed
+from 'SHA-1' to 'SHA-256', see the `Set the secure hash algorithm` section
+in the [ADFS administrator guide](docs/en/adfs.md) for what setting to change.
+
+#### Absolute paths for the SP and IDP certificates and key files
+
+An issue was fixed where absolute paths couldn't be used for certificates and
+keys in YAML configuration.
+
+### Changes to the LDAP integration
+
+ - Better LDAP group syncing that isn't as likely to crash and leave the database
+   in an unknown state.
+ - Removes LDAP group mappings when a security group is deleted
+ - Minor UI changes for the login form
+ - Moved the AD fields in the CMS to a separate tab and made them read-only.
+ - Member "last synced" time is now showing the correct date and time.
+ - Adding a script to rotate LDAP binding credentials for system administrators
+ - Samba has been confirmed to work
+ - Automatic syncing of LDAP groups and users via the `LDAPAllSyncJob` build task
+ - Users can now reset their AD password via the "I've lost my password" on the
+   login form.
+ - Updated documentation
+
+The LDAP functionality that does changes to a users data (i.e. user password
```

Comment:
How about "LDAP functionality that modifies user data (i.e. password reset) will require binding credentials with LDAP write access"?